android/keyframes/src/main/java/com/facebook/keyframes/KeyframesDrawableAnimationCallback.java

```diff
@@ -105,6 +105,17 @@ public void start() {
   }
 
   /**
+   * Starts the animation and plays it once
+   */
+  public void playOnce() {
+    mStopAtLoopEnd = true;
+    mStartTimeMillis = 0;
+    mCurrentLoopNumber = 0;
```

Comment:
0 should be correct. `currentLoopNumber` starts with 0 and if I set `mCurrentLoopNumber` to -1 it will stop right away cause its smaller and `loopHasEnded` will be `true` right away.